src/components/molecules/GitHubRepositoriesFeed.js

```diff
@@ -0,0 +1,77 @@
+import React, {Component} from "react";
+
+import Card from "./Card";
+import Tag from "../atoms/Tag";
+import config from "../../config";
+
+import "../../css/molecules/githubrepositoriesfeed.css";
+
+class GitHubRepositoriesFeed extends Component {
+    state = {
+        github_repositories: []
+    };
+
+    constructor(props) {
+        super(props);
+
+        this.user = props.user;
+        this.account = props.account;
+    }
+
+    async getGitHubProjects(user) {
+        const api = config["github-api"];
+
+        // const request = await fetch(`${api}/user/v1/github-repos/${user}`);
+        // const data = await request.json();
+
+        // temp data until endpoint is real
+        const data = [{
+            id: "1",
+            name: "Advent of Code",
+            updated_at: "2019-12-02T19:48:06Z",
+            description: "My answers to the Advent of Code challenges.",
+            url: "https://github.com/LamboCreeper/Advent-of-Code",
+            stars: 2,
+            watches: 1,
+            forks: 0,
+            language: "Python"
+        }];
+
+        this.setState({
+            github_repositories: data.sort((a, b) => {
+                return new Date(a.updated_at) > new Date(b.updated_at) ? -1 : 1
+            }).slice(0, 5)
+        });
```

Comment:
I would suggest looking at [React-Redux](https://react-redux.js.org/) as a possible addition before this project grows too large.